java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -220,6 +221,7 @@ public static Object[] transactionTestResult() {
             Set.of("baz"), // smembers(key7)
             new Boolean[] {true, false}, // smismembmer(key7, new String[] {"baz", "foo"})
             Set.of("baz"), // sinter(new String[] { key7, key7 })
+            Set.of(), // sdiff(new String[] {key7, key7})
```

Comment:
a slightly better test would be `sdiff(new String[] {key7, emptyKey})` to return the contents of `key7`.